climada_petals/entity/exposures/openstreetmap/osm_dataloader.py

```diff
@@ -0,0 +1,945 @@
+        'healthcare' : {
+            'osm_keys' : ['amenity','building','healthcare','name'],
+            'osm_query' : """amenity='hospital' or healthcare='hospital' or 
+                             building='hospital' or building='clinic' or
+                             amenity='clinic' or healthcare='clinic' or 
+                             amenity='doctors' or healthcare='doctors'
+                             """}, # amenity='dentist' or healthcare='dentist' or amenity='pharmacy' or amenity='nursing_home' or healthcare='*'
+        'water' : {
+            'osm_keys' : ['man_made','pump','pipeline','emergency','name'],
+            'osm_query' : """man_made='water_well' or man_made='water_works' or
+                             man_made='water_tower' or
+                             man_made='reservoir_covered' or 
+                             landuse='reservoir' or
+                             (man_made='pipeline' and substance='water') or
+                             (pipeline='substation' and substance='water') or
+                             pump='powered' or pump='manual' or pump='yes' or
+                             emergency='fire_hydrant' or
+                             (man_made='storage_tank' and content='water')"""},
+        'telecom' : {
+            'osm_keys' : ['man_made','tower_type','telecom',
+                          'communication_mobile_phone','name'],
+            'osm_query' : """tower_type='communication' or man_made='mast' or
+                             communication_mobile_phone='*' or
+                             telecom='antenna' or
+                             telecom='poles' or communication='pole' or
+                             telecom='central_office' or 
+                             telecom='street_cabinet' or
+                             telecom='exchange' or telecom='data_center' or
+                             telecom='distribution_point' or 
+                             telecom='connection_point' or
+                             telecom='line' or communication='line' or
+                             utility='telecom'"""},
+        'road' :  {
+            'osm_keys' : ['highway','man_made','public_transport','bus','name'],
+            'osm_query' : """highway='motorway' or highway='motorway_link' or
+                             highway='trunk' or highway='trunk_link' or
+                             highway='primary' or highway='primary_link' or
+                             highway='secondary' or highway='secondary_link' or
+                             highway='tertiary' or highway='tertiary_link' or
+                             highway='residential' or highway='road' or
+                             highway='service' or highway='unclassified' or
+                             highway='traffic_signals' or
+                             (public_transport='*' and bus='yes') or
+                             man_made='bridge' or man_made='tunnel'"""},
```

Comment:
will this include other, non-road bridges and tunnels?

Reply:
Well spotted! I updated the conditions to only report those tunnels & bridges that also have a highway tag.